pkg/webui/console/components/api-key-form/validation-schema.js

```diff
@@ -20,6 +20,13 @@ const validationSchema = Yup.object().shape({
     .min(2, Yup.passValues(sharedMessages.validateTooShort))
     .max(50, Yup.passValues(sharedMessages.validateTooLong)),
   rights: Yup.array().min(1, sharedMessages.validateRights),
+  expires_at: Yup.lazy(value => {
+    if (!Boolean(value)) {
+      return Yup.string()
+    }
+
+    return Yup.date()
+  }),
```

Comment:
```suggestion
  expires_at: Yup.string().nullable(),
  }),
```

If you use the encode/decode props, you'll just have two possible types: string (a string representing the date, since this is what the backend expects) and null. Please add additional validation though to prevent users from inputting dates in the past.